src/components/DocumentationTopic.vue

```diff
@@ -129,7 +129,6 @@
               :source="remoteSource"
               :sections="primaryContentSectionsSanitized"
             />
-            <ViewMore v-if="shouldShowViewMoreLink" :url="viewMoreLink" />
```

Comment:
I may be misunderstanding things, but I'm wondering if you could simplify a lot of these changes by just conditionally rendering the relationships section above this and continuing to keep things in the "minimized-container".

As long as you make sure that you have one `<Relationships v-if="enableMinimized"...` here and one `<Relationships v-if="!enableMinimized..."` below, would that work and be a cleaner way of including this section in that mode?

I have a feeling you may have already looked into that approach, but I'm just curious. As much as possible, it would be nice to have the parent component dictate the layout for its children as opposed to having to pass the `enableMinimized` flags to lots of child components, where it doesn't really make sense in isolation for them.